src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -50,7 +50,7 @@ public static function escapeTrailingBackslash($text)
         if ('\\' === substr($text, -1)) {
             $len = strlen($text);
             $text = rtrim($text, '\\');
-            $text .= str_repeat('<<', $len - strlen($text));
+            $text .= str_repeat('\0', $len - strlen($text));
```

Comment:
tested with `"\0"` on 3.0.4 and seems to work, thanks @javiereguiluz for picking this up, much appreciated :+1:  :)